sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProvider.java

```diff
@@ -115,14 +115,10 @@ protected SchemaTransform from(KafkaReadSchemaTransformConfiguration configurati
 
     String format = configuration.getFormat();
 
-    if (format == null || format.isEmpty()) {
-      format = "raw";
-    }
-
-    if (format.equalsIgnoreCase("raw")) {
-      if (inputSchema != null && !inputSchema.isEmpty()) {
+    if (format != null && format.equals("RAW")) {
+      if (inputSchema != null) {
         throw new IllegalArgumentException(
-            "To read from Kafka in raw format, you can't provide a schema.");
+            "To read from Kafka in RAW format, you can't provide a schema.");
```

Comment:
are there other error scenarios that we should include?  I'm trying to think about this one

Reply:
I couldn't think of any other but I'm open to suggestions